generators/client/templates/angular/src/main/webapp/app/app.module.ts.ejs

```diff
@@ -71,6 +70,7 @@ import {
         <%=angularXAppName%>HomeModule,
         <%_ if (authenticationType !== 'oauth2') { _%>
         <%=angularXAppName%>AccountModule,
+        <%=angularXAppName%>AppRoutingModule,
```

Comment:
@lionelpnt Did you add by mistake the AppRoutingModule import in <> ouath2 condition?

Reply:
Yes it should be below the EntityModule import